lib/pry_debugging.rb

```diff
@@ -1,4 +1,4 @@
 def plus_two(num)
-	num + 2
-	num
-end
\ No newline at end of file
+	sum = num + 2
+	sum
```

Comment:
This would also work. 
```suggestion
	num + 2
```